apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V27__Create_BeneficiaryMonthlytable.sql

```diff
@@ -0,0 +1,142 @@
+    "partCContractNumberAugId", "partCPbpNumberAugId", "partCPlanTypeAugCode",
+    "partDContractNumberAugId", "partDPbpNumberAugId", "partDSegmentNumberAugId",
+    "partDRetireeDrugSubsidyAugInd", "medicaidDualEligibilityAugCode", "partDLowIncomeCostShareGroupAugCode"
+    FROM "Beneficiaries"
+    WHERE "beneEnrollmentReferenceYear" is not null;
+
+INSERT INTO "BeneficiaryMonthly" 
+    SELECT  "beneficiaryId", TO_DATE(CONCAT("beneEnrollmentReferenceYear", '/09/01'), 'YYYY/MM/DD') as "yearMonth", "fipsStateCntySeptCode",
+    "medicareStatusSeptCode", "entitlementBuyInSeptInd", "hmoIndicatorSeptInd",
+    "partCContractNumberSeptId", "partCPbpNumberSeptId", "partCPlanTypeSeptCode",
+    "partDContractNumberSeptId", "partDPbpNumberSeptId", "partDSegmentNumberSeptId",
+    "partDRetireeDrugSubsidySeptInd", "medicaidDualEligibilitySeptCode", "partDLowIncomeCostShareGroupSeptCode"
+    FROM "Beneficiaries"
+    WHERE "beneEnrollmentReferenceYear" is not null;
+
+INSERT INTO "BeneficiaryMonthly" 
+   SELECT  "beneficiaryId", TO_DATE(CONCAT("beneEnrollmentReferenceYear", '/10/01'), 'YYYY/MM/DD') as "yearMonth", "fipsStateCntyOctCode",
+    "medicareStatusOctCode", "entitlementBuyInOctInd", "hmoIndicatorOctInd",
+    "partCContractNumberOctId", "partCPbpNumberOctId", "partCPlanTypeOctCode",
+    "partDContractNumberOctId", "partDPbpNumberOctId", "partDSegmentNumberOctId",
+    "partDRetireeDrugSubsidyOctInd", "medicaidDualEligibilityOctCode", "partDLowIncomeCostShareGroupOctCode"
+    FROM "Beneficiaries"
+    WHERE "beneEnrollmentReferenceYear" is not null;  
+
+INSERT INTO "BeneficiaryMonthly" 
+    SELECT  "beneficiaryId", TO_DATE(CONCAT("beneEnrollmentReferenceYear", '/11/01'), 'YYYY/MM/DD') as "yearMonth", "fipsStateCntyNovCode",
+    "medicareStatusNovCode", "entitlementBuyInNovInd", "hmoIndicatorNovInd",
+    "partCContractNumberNovId", "partCPbpNumberNovId", "partCPlanTypeNovCode",
+    "partDContractNumberNovId", "partDPbpNumberNovId", "partDSegmentNumberNovId",
+    "partDRetireeDrugSubsidyNovInd", "medicaidDualEligibilityNovCode", "partDLowIncomeCostShareGroupNovCode"
+    FROM "Beneficiaries"
+    WHERE "beneEnrollmentReferenceYear" is not null;  
+
+INSERT INTO "BeneficiaryMonthly" 
+    SELECT  "beneficiaryId", TO_DATE(CONCAT("beneEnrollmentReferenceYear", '/12/01'), 'YYYY/MM/DD') as "yearMonth", "fipsStateCntyDecCode",
+    "medicareStatusDecCode", "entitlementBuyInDecInd", "hmoIndicatorDecInd",
+    "partCContractNumberDecId", "partCPbpNumberDecId", "partCPlanTypeDecCode",
+    "partDContractNumberDecId", "partDPbpNumberDecId", "partDSegmentNumberDecId",
+    "partDRetireeDrugSubsidyDecInd", "medicaidDualEligibilityDecCode", "partDLowIncomeCostShareGroupDecCode"
+    FROM "Beneficiaries"
+    WHERE "beneEnrollmentReferenceYear" is not null;
+
+alter table "BeneficiaryMonthly" 
+  add constraint "BeneficiaryMonthly_parentClaim_to_Beneficiary" foreign key ("parentBeneficiary") 
```

Comment:
You have a copy-paste error here to fix:
```suggestion
  add constraint "BeneficiaryMonthly_parentBeneficiary_to_Beneficiary" foreign key ("parentBeneficiary") 
```

Reply:
Changed

Reply:
Nice eyes. Upset I didnt catch that. TY.

Reply:
I've looked at this... five times now, and it's the first time I noticed it, so... :shrug: .